### DISCUSSION TITLE
Script worked perfectly at 5:50am EDT this morning

### DISCUSSION THREAD

**Discussion by schmeed** [jdholtz/auto-southwest-check-in#54] [General]
I just wanted to pass along that the script ran perfectly with 3 flights inside config.json.

Ubuntu 20.0.4.2 LTS
Pip 22.0.2
Python 3.10.6
Google Chrome 110.0.5481.177
undetected-chromedriver 3.1.7
Auto-Southwest Check-In v2.0

Thank you @jdholtz!!!!!!  Where can I send you a donation for your work on this?

Output from pip freeze:
apprise==1.2.1
apturl==0.5.2
async-generator==1.10
attrs==22.2.0
blinker==1.4
Brlapi==0.8.3
Brotli==1.0.9
certifi==2022.12.7
cffi==1.15.1
chardet==4.0.0
charset-normalizer==3.0.1
click==8.0.3
colorama==0.4.4
command-not-found==0.3
cryptography==39.0.2
cupshelpers==1.0
dbus-python==1.2.18
defer==1.0.6
distro==1.7.0
distro-info===1.1build1
exceptiongroup==1.1.0
h11==0.14.0
h2==4.1.0
hpack==4.0.0
httplib2==0.20.2
hyperframe==6.0.1
idna==3.3
importlib-metadata==4.6.4
jeepney==0.7.1
kaitaistruct==0.10
keyring==23.5.0
language-selector==0.1
launchpadlib==1.10.16
lazr.restfulclient==0.14.4
lazr.uri==1.0.6
louis==3.20.0
macaroonbakery==1.3.1
Markdown==3.4.1
more-itertools==8.10.0
netifaces==0.11.0
oauthlib==3.2.0
olefile==0.46
outcome==1.2.0
pexpect==4.8.0
Pillow==9.0.1
protobuf==3.12.4
ptyprocess==0.7.0
pyasn1==0.4.8
pycairo==1.20.1
pycparser==2.21
pycups==2.0.1
PyGObject==3.42.1
PyJWT==2.3.0
pymacaroons==0.13.0
PyNaCl==1.5.0
pyOpenSSL==23.0.0
pyparsing==2.4.7
pyRFC3339==1.1
PySocks==1.7.1
python-apt==2.4.0+ubuntu1
python-dateutil==2.8.1
python-debian===0.1.43ubuntu1
pytz==2022.7.1
pyxdg==0.27
PyYAML==5.4.1
reportlab==3.6.8
requests==2.28.2
requests-oauthlib==1.3.1
SecretStorage==3.3.1
selenium==4.8.0
selenium-wire==5.1.0
six==1.16.0
sniffio==1.3.0
sortedcontainers==2.4.0
systemd-python==234
trio==0.22.0
trio-websocket==0.9.2
ubuntu-advantage-tools==27.9
ubuntu-drivers-common==0.0.0
ufw==0.36.1
unattended-upgrades==0.1
undetected-chromedriver==3.1.7
urllib3==1.26.5
wadllib==1.3.6
websockets==10.4
wsproto==1.2.0
xdg==5
xkit==0.0.0
zipp==1.0.0
zstandard==0.20.0

**Comment by jdholtz** [comment]
Glad to hear you got it to work! Thanks for providing information about your environment as that will be helpful in knowing where Chrome v110 fails and where it works.

> Where can I send you a donation for your work on this?

I appreciate your willingness to donate! I set up a GitHub Sponsors page and that should be approved within the next few days. That would be the best place to send it.

**Reply by schmeed** [reply]
@jdholtz , your sponsor page is active and I just made a donation.  I encourage everyone else to kick in as well.  This script is awesome and we need to show our gratitude.

**Reply by TeSoad90** [reply]
agreed ^
